src/app.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
```

Comment:
Duplicate file? This is generated from the TS, no?

Reply:
yes, this should be the compiled file from TS.